### PR TITLE
change border

### PR DESCRIPTION
**Pull request by aleixcam**
pls accept pull req

very important work

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,10 +6,13 @@
...
-        body{
+        body {
+            min-height: calc(100vh - 60px);
+            margin: 0;
+            padding: 20px;
             background-color: #fa0;
-            color:white;
-            border: 10px solid skyblue;
+            color: white;
+            border: 10px solid greenyellow;
```
Comment (Pacoscript) [inline index.html]: quien te mando poner greenyellow?

Comment (aleixcam) [reply index.html]: el profe

File: index.html [hunk 2]
```diff
@@ -6,10 +6,13 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Hello World</title>
     <style>
```
Comment (Pacoscript) [inline index.html]: hey move styles to file, please

Comment (aleixcam) [reply index.html]: u 2

Comment (Pacoscript) [reply index.html]: don´t worry, i´ll create de style file

Comment (aleixcam) [reply index.html]: worrysome

**Pull request merged**